### PR TITLE
workflow statistics

### PR DESCRIPTION
**Pull request by azopat**

### CODE REVIEW

File: argo/README.md [hunk 1]
```diff
@@ -53,3 +53,53 @@ Install the listener
 ```bash
 kubectl apply -n argo-events -f minio-eventsource.yml -f minio-event-sensor.yml
```
Comment (yardenshoham) [inline argo/README.md:54]:
```suggestion
kubectl apply -f minio-eventsource.yml -f minio-event-sensor.yml
```
Please update the README commands to use the `argo` namespace.

Comment (yardenshoham) [reply argo/README.md:54]: I'm referring to lines `47-48`.

**Review by yardenshoham — CHANGES_REQUESTED** [review]

**Comment by azopat** [comment]
done

**Comment by yardenshoham** [comment]
@azopat also lines `47-48`.

**Review by yardenshoham — APPROVED** [review]

**Pull request merged**